_data/credits.yml

```diff
@@ -0,0 +1,75 @@
+active_core_devs: [
+  "Perttu Ahola (celeron55) &#x3C;<a href='mailto:&#x63;&#x65;&#x6C;&#x65;&#x72;&#x6F;&#x6E;&#x35;&#x35;&#x40;&#x67;&#x6D;&#x61;&#x69;&#x6C;&#x2E;&#x63;&#x6F;&#x6D;' title='&#x63;&#x65;&#x6C;&#x65;&#x72;&#x6F;&#x6E;&#x35;&#x35;&#x40;&#x67;&#x6D;&#x61;&#x69;&#x6C;&#x2E;&#x63;&#x6F;&#x6D;'>&#x63;&#x65;&#x6C;&#x65;&#x72;&#x6F;&#x6E;&#x35;&#x35;&#x40;&#x67;&#x6D;&#x61;&#x69;&#x6C;&#x2E;&#x63;&#x6F;&#x6D;</a>&#x3E;",
```

Comment:
Why the obfuscation? Anything scraping the website (and using a correct HTML parser) will obtain these addresses just fine. I also don't like embedding HTML here; why not use a clean YAML instead and generate the proper HTML in the template?

Reply:
I haven't changed this, it was encoded like this before

Reply:
I bet you could script this but also it doesn't matter

Reply:
Done